### PR TITLE
deployment: prepare v0.5.0

### PR DESCRIPTION
**Pull request by desimone**
## Summary

Re-organized docs a bit. Community is now a sub-setion of `docs`, and broke out configuration options and examples as their own subsection.

https://5dc73df5bde5f7000877cc24--pomerium-docs.netlify.com

## Related issues

**Checklist**:

**Before tagging**

- [x] Bump version
- [x] Update docs to have a new version in config.js and in netlify
- [x] Review and update examples 
- [x] Review and update dockerfiles 
- [x] Review and update docs
- [x] Review and update changelog
- [x] Review and update upgrading guide

**After tagging**
- [ ] Update github release docs
- [ ] Set `latest` branch to track the new release
- [ ] Publish tagged version of documentation
- [ ] Review and update helm-charts (merge after tag) 
- [ ] Change docker examples back to `master` on master.

### CODE REVIEW

**Comment by victornoel** [comment]
@desimone if you can wait, I will do some tests with the master branch on the weekend :)

**Comment by victornoel** [comment]
@desimone the CI does not seem to build the latest docker image anymore from master, so I can't test :/

**Comment by victornoel** [comment]
@desimone sorry, my bad, the tag is `master`

**Comment by victornoel** [comment]
@desimone so first test I do, when I connect to my app:
- I get disconnected (as expected since session handling changed)
- redirected to IdP
- I login
- and I get redirected to IdP in loops

Authenticate service is logging:
```
{"level":"info","fwd_ip":["77.141.109.228"],"ip":"10.242.1.28","user_agent":"Mozilla/5.0 (X11; Linux x86_64; rv:70.0) Gecko/20100101 Firefox/70.0","req_id":"2fb05e78-a879-a290-0a3e-8ad5e00af8a9","error":"internal/sessions: session is malformed","time":"2019-11-10T10:06:56Z","message":"authenticate: verify session"}
```

If I remove all the old cookies related to pomerium, then it works.

I suppose we are missing some mechanism to forget all previous cookies from previous pomerium version.

**Comment by victornoel** [comment]
@desimone could you remind me how to I disable redirect on authentication error from the proxy as discussed in #281?

File: docs/docs/reference/programmatic-access.md [hunk 1]
```diff
@@ -7,63 +7,81 @@ description: >-
...
+
+Your application should handle token expiration. If the session expires before work is done, the identity provider issued `refresh_token` can be used to create a new valid session.
 
-It's not as bad as it sounds. Please see the following minimal but complete examples.
+Also, your script or application should anticipate the possibility that a granted `refresh_token` may stop working. For example, a refresh token might stop working if the underlying user changes passwords, revokes access, or if the administrator removes rotates or deletes the OAuth Client ID.
 
-### Python
+## High level workflow
+
+The application interacting with Pomerium must manage the following workflow flow. Consider the following example where a script or program desires delegated, programmatic access to the domain `httpbin.corp.domain.example`:
```
Comment (travisgroth) [inline docs/docs/reference/programmatic-access.md]: >workflow flow

File: docs/docs/reference/programmatic-access.md [hunk 2]
```diff
@@ -7,63 +7,81 @@ description: >-
...
+
+## Example Code
+
+Please consider see the following minimal but complete python example.
 
 ```bash
-python scripts/programmatic_access.py --client-secret REPLACE_ME \
-    --client-id 851877082059-85tfqg9hlm8j9km5d9uripd0dvk72mvk.apps.googleusercontent.com \
-    --pomerium-client-id 851877082059-bfgkpj09noog7as3gpc3t7r6n9sjbgs6.apps.googleusercontent.com
+python3 scripts/programmatic_access.py \
```
Comment (travisgroth) [inline docs/docs/reference/programmatic-access.md]: 👍

File: docs/docs/reference/programmatic-access.md (hunk 1 above)
Comment (desimone) [reply docs/docs/reference/programmatic-access.md]: Thanks good catch.

**Pull request merged**